packages/orderbook/test/js_api/gui.test.ts

```
@@ -607,19 +608,19 @@ describe('Rain Orderbook JS API Package Bindgen Tests - Gui', async function ()
 			const allFieldDefinitions = gui.getAllFieldDefinitions();
 			gui.saveFieldValue('binding-1', {
 				isPreset: true,
-				value: allFieldDefinitions[0].presets[0].id
+				value: allFieldDefinitions[0].presets![0].id
 			});
-			assert.deepEqual(gui.getFieldValue('binding-1'), allFieldDefinitions[0].presets[0]);
+			assert.deepEqual(gui.getFieldValue('binding-1'), allFieldDefinitions[0].presets![0]);
 			gui.saveFieldValue('binding-1', {
 				isPreset: true,
-				value: allFieldDefinitions[0].presets[1].id
+				value: allFieldDefinitions[0].presets![1].id
```

Biome (1.9.4)

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)